plugin/pkg/proxy/connect.go

```diff
@@ -152,9 +152,9 @@ func (p *Proxy) Connect(ctx context.Context, state request.Request, opts Options
 		rc = strconv.Itoa(ret.Rcode)
 	}
 
-	RequestCount.WithLabelValues(p.addr).Add(1)
-	RcodeCount.WithLabelValues(rc, p.addr).Add(1)
-	RequestDuration.WithLabelValues(p.addr, rc).Observe(time.Since(start).Seconds())
+	RequestCount.WithLabelValues(p.proxyName, p.addr).Add(1)
+	RcodeCount.WithLabelValues(p.proxyName, p.addr, rc).Add(1)
+	RequestDuration.WithLabelValues(p.proxyName, p.addr, rc).Observe(time.Since(start).Seconds())
```

Comment:
Same with the proxy metrics. While we're here let's fix them to be unexported.

```suggestion
	requestCount.WithLabelValues(p.proxyName, p.addr).Add(1)
	rcodeCount.WithLabelValues(p.proxyName, p.addr, rc).Add(1)
	requestDuration.WithLabelValues(p.proxyName, p.addr, rc).Observe(time.Since(start).Seconds())
```

Reply:
Have unexported all metrics defined within the `proxy` plugin.